packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -74,6 +74,15 @@ export interface BranchRebaseResult<TChange> {
 	 * Details about how the commits on the source branch changed
 	 */
 	readonly commits: RebasedCommits<TChange>;
+	/**
+	 * The length of the source branch after the rebase.
+	 */
+	readonly sourceBranchLength?: number;
```

Comment:
I would be good to update the tests in `src\test\rebase\rebaseBranch.spec.ts` so they expect the correct returned values.
You may want to wait until we settle the question of what these values should be though, since we're still unclear on that right now.